assignment-8-flipkart-page-1/index.html

```diff
@@ -0,0 +1,799 @@
+        </div>
+
+        <div class="storage">
+          <div>
+            <p>OPERATING SYSTEM</p>
+          </div>
+          <div>
+            <a href=""><img src="img/downarrow.svg" width="7"></a>
+          </div>
+        </div>
+
+        <div class="storage">
+          <div>
+            <p>NETWORK TYPE</p>
+          </div>
+          <div">
+            <a href=""><img src="img/downarrow.svg" width="7"></a>
+        </div>
+      </div>
+
+      </div>
+
+      <div class="mainbox">
+        <div class="arrowContainer">
+          <div class="arrowtext">
+            <a href="">Home</a>
+          </div>
+          <div class="arrowtext">
+            <a href=""><img class="arrow" src="img/next.svg" width="7"></a>
+          </div>
+          <div class="arrowtext">
+            <a href="">Mobiles & Accessories</a>
+          </div>
+          <div class="arrowtext">
+            <a href=""><img class="arrow" src="img/next.svg" width="7"></a>
+          </div>
+          <div class="arrowtext">
+            <a href="">Mobiles</a>
+          </div>
+        </div>
+
+        <div class="txt">
+          <a class="fromMI">Latest from MI : </a><a class="redmi" href="">Redmi Go</a>
+        </div>
+
+        <div class="txt1">
+          <a class="txt11">MI Mobiles</a><a class="txt12" href="">(Showing 1 – 24 products of 247 products)</a>
+        </div>
+
+        <div class="sort">
+          <div>
+            <p>Sort By</p>
+          </div>
+          <div>
+            <p class="popularity">Popularity</p>
+          </div>
+          <div>
+            <p>Price--Low to High</p>
+          </div>
+          <div>
+            <p>Price--High to Low</p>
+          </div>
+          <div>
+            <p>Newest First</p>
+          </div>
+        </div>
+
+        <div class="note7">
+          <div class="note7img">
+            <img class="mobile1" src="img/note7.jpeg" width="80">
+            <form class="addtocompare">
+              <input type="checkbox" id="add" name="compare" value="compare">
+              <label for="add">Add to compare</label><br>
+            </form>
+          </div>
+
+          <div>
+            <a href=""><img class="heart" src="img/heart.svg" width="13"></a>
+          </div>
+
+          <div class="specifications">
+            <a>Redmi Note 7 Pro (Space Black, 128 GB)</a>
+            <div class="title">
+              <div class="rating_dropdown">
+                <a class="rating" href="">4.5 &#9733</a>
+                <div class="rating_content">
+                  <div class="rt1">
+                    <a class="r1">4.5 &#9733</a><br>
+                    <a class="r2">2,68,700 Ratings &</a><br>
+                    <a class="r2">22,596 Reviews</a>
+                  </div>
+                  <div class="star">
+                    <div class="s1">
+                      <P>5 &#9733</P>
+                    </div>
+                    <div class="st2">
+                      <!-- //// -->
```

Comment:
what is the purpose of this empty div? Is it for spacing? It would be good to add comments in cases like this. In general do add meaningful comments so someone reading the code will get a basic understanding.